recipe/meta.yaml

```diff
@@ -1,9 +1,8 @@
-{% set name = "cuda-version" %}
-{% set version = "12.0.0" %}
-{% set major_version = version.split('.')[0]|int %}
+{% set version = major_minor_ver|default("12.0") %}
```

Comment:
Do we want a default here? I vote no. I'd prefer a hard error.

Reply:
Yes, in conda-build's first (few?) passes the Jinja variables are not fully resolved yet, so without a default it'd be a parser error.

Reply:
Yeah exactly what Leo said. Conda-build needs to do a few passes and earlier passes need to start with something

That said, there's no real risk of that default being used. So don't think we need to worry about it too much

Reply:
Found a better solution in the revised recipe that is hopefully clearer about the parsing stages